routes/exchange.go

```diff
@@ -1617,48 +1618,72 @@ func (fes *APIServer) GetPostsForFollowFeedForPublicKey(bav *lib.UtxoView, start
 		return nil, errors.Wrapf(err, "GetPostsForFollowFeedForPublicKey: Problem filtering out restricted public keys: ")
 	}
 
-	minTimestampNanos := uint64(time.Now().UTC().AddDate(0, 0, -2).UnixNano()) // two days ago
-	// For each of these pub keys, get their posts, and load them into the view too
-	for _, followedPubKey := range filteredPubKeysMap {
-
-		_, dbPostAndCommentHashes, _, err := lib.DBGetAllPostsAndCommentsForPublicKeyOrderedByTimestamp(
-			bav.Handle, fes.blockchain.Snapshot(), followedPubKey, false /*fetchEntries*/, minTimestampNanos, 0, /*maxTimestampNanos*/
-		)
-		if err != nil {
-			return nil, errors.Wrapf(err, "GetPostsForFollowFeedForPublicKey: Problem fetching PostEntry's from db: ")
+	var postEntriesForFollowFeed []*lib.PostEntry
+	baseDaysToSubtract := 4
+	maxDaysToSubtractExponent := 5 // 4^5 = 1024 days
+	for daysToSubtractExponent := 1; daysToSubtractExponent < 6; daysToSubtractExponent++ {
+		daysToSubtract := int(math.Pow(float64(baseDaysToSubtract), float64(daysToSubtractExponent))) // baseDaysToSubtract ^ daysToSubtractExponent
+		postsFetched := len(postEntriesForFollowFeed)
+
+		minTimestampNanos := uint64(time.Now().UTC().AddDate(0, 0, -daysToSubtract).UnixNano()) // baseDaysToSubtract ^ daysToSubtractExponent days ago
+		maxTimestampNanos := uint64(0)
+
+		if postsFetched > 0 {
+			maxTimestampNanos = postEntriesForFollowFeed[postsFetched-1].TimestampNanos                                  // set to last fetched
```

Comment:
can you use the `minTimestampNanos` from the previous loop here? that would prevent needing to sort the entries on each loop, and would make this more accurate (i.e. not rescanning timestamps that were already scanned)